### PR TITLE
update scheduled_sampling/README.en.md

### LINKED ISSUE #712: Translation Plan-Scheduled Sampling-汉译英
## 任务背景
针对v2版本，目前我们已有很多中文使用文档，但仍有大量的汉译英工作及少量英译汉工作没有完成，我们希望将中英文版本补充完全，提升用户的阅读体验，也有益于国外用户在使用时查阅文档。
此外，我们的模型库Models，仍有大部分英译汉和汉译英的工作，亟待完成。

## 任务描述
1. 需要做什么：
把“Scheduled Sampling”**汉译英**

2. 地址
- 中文地址： [Github地址](https://github.com/PaddlePaddle/models/blob/develop/scheduled_sampling/README.md)，[网页地址](http://www.paddlepaddle.org/docs/develop/models/scheduled_sampling/README.html)
- 需要翻译的英文地址，暂无，可在中文版所在github目录下创建md文档

## 如何认领
我们欢迎大家积极认领，内部小伙伴请通过self assign的方式认领，外部小伙伴可通过在本issue下方comment认领。

## 注意事项
上交初稿的截止日期是2018年4月15日
上交后，PaddlePaddle的RD会为大家review
请于review后的5个工作日之内完成修改

## 相关信息
所有需要翻译的文档：
V2使用文档：
https://shimo.im/sheet/PqK407VYm80Ju3dw
Models:
https://shimo.im/sheet/6XTZnRQEUS0uC1Ut

### PR DESCRIPTION
**Pull request by vienous**
fix https://github.com/PaddlePaddle/models/issues/712

### CODE REVIEW

**Comment by shanyi15** [comment]
您好
CI测试未通过，失败原因
Trim Trailing Whitespace.................................................Failed
请参考[翻译常见问题](https://shimo.im/docs/zMRNcbMM3jA58ZJg)的第5个问题尝试解决，谢谢！

File: scheduled_sampling/READMD.en.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+
```
Comment (shanyi15) [inline scheduled_sampling/READMD.en.md]: 文件名写错，应为README.en.md

File: scheduled_sampling/READMD.en.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+Run these examples of programs in this directory require PaddlePaddle v0.10.0 . If your PaddlePaddle installation is lower than this requirement, please follow the instructions in the [installation documentation](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html) to update the PaddlePaddle version.
```
Comment (shanyi15) [inline scheduled_sampling/READMD.en.md]: 关于版本信息，这里有一个参考："The program under this directory depends on PaddlePaddle v0.10.0 or upper. update it following this instruction if your PaddlePaddle version is lower."

File: scheduled_sampling/READMD.en.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+ - Linear decay：$\epsilon_i=max(\epsilon,k-c*i)$，Where $\epsilon$ limits the minimum value of $\epsilon_i$, $k$ and $c$ control the magnitude of linear decay.
+
+ - Exponential decay：$\epsilon_i=k^i$，Where $0<k<1$ and $k$ control the magnitude of the exponential decay.
+
+ - Reverse Sigmoid decay：$\epsilon_i=k/(k+exp(i/k))$，Where $k>1$, $k$ also controls the magnitude of attenuation.
+
+Figure 1 shows the attenuation curves for these three methods.
+
+<p align="center">
+<img src="images/decay.jpg" width="50%" align="center"><br>
```
Comment (shanyi15) [inline scheduled_sampling/READMD.en.md]: 这里请将图片换成英文版的，并重新上传一份英文版图片配到文档里，路径同中文版图片。

File: scheduled_sampling/READMD.en.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+Run these examples of programs in this directory require PaddlePaddle v0.10.0 . If your PaddlePaddle installation is lower than this requirement, please follow the instructions in the [installation documentation](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html) to update the PaddlePaddle version.
+
+---
+
+# Scheduled Sampling
+
+## Introduction
+
+The generate target of sequence generate task is to maximize the probability of target sequence when a given source is input. During the training, the model input real elements as decoders to each step, then maximize the probability of next element. During the generation, the element created in the previous decoder step is used as current input, consequently generate the next element. In this case, It can been seen probability distribution in training stage and generation stage are not consistent.
+
```
Comment (shanyi15) [inline scheduled_sampling/READMD.en.md]: It can be seen, i要小写，而且seen 后面最好加一个that

**Review by shanyi15 — COMMENTED** [review]
请先修改格式以通过CI测试，然后根据现有的意见修改，谢谢！

**Comment by shanyi15** [comment]
您好，之前几个还是同一个问题CI挂了，建议用sublime编辑器，我最近提的几个文档，都是亲测可行的
https://shimo.im/docs/zMRNcbMM3jA58ZJg
这里第5个问题的第2个方法

**Comment by shanyi15** [comment]
reviewer please note that: before you merge, please select `squash and merge` to simplify blog.

**Comment by vienous** [comment]
最后一个可以了啊，我用的第一个方法

File: scheduled_sampling/README.en.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
+﻿
+The program under this directory depends on PaddlePaddle v0.10.0 or upper. update it following this [instruction](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html)  if your PaddlePaddle version is lower.
```
Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: update it following this [instruction] -> In order to update it, follow these instructions

File: scheduled_sampling/README.en.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+
+---
+
+# Scheduled Sampling
+
+## Introduction
+
+The generate target of sequence generate task is to maximize the probability of target sequence when a given source is input. During the training, the model input real elements as decoders to each step, then maximize the probability of next element. During the generation, the element created in the previous decoder step is used as current input, consequently generate the next element. In this case, it can been seen that probability distribution in training stage and generation stage are not consistent.
```
Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: generate target -> **generated** target

Comment (abhinavarora) [reply scheduled_sampling/README.en.md]: sequence generate task -> sequence **generation** task

Comment (abhinavarora) [reply scheduled_sampling/README.en.md]: During **the** training, -> During training,

Comment (abhinavarora) [reply scheduled_sampling/README.en.md]: maximize -> maximizes

Comment (abhinavarora) [reply scheduled_sampling/README.en.md]: current input, consequently generate the next element. -> current input to consequently generate the next element.

Comment (abhinavarora) [reply scheduled_sampling/README.en.md]: that probability distribution  ->  that **the** probability distribution

File: scheduled_sampling/README.en.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+
+Scheduled Sampling \[[1](#参考文献)\] is a method to solve the inconsistent distribution of input data during training and generation.In the early stage of training, this method mainly uses the real elements in the target sequence as the decoder input, and can quickly guide the model from a randomly initialized state to a reasonable state. As training progresses, the method will gradually increase the use of the generated element as decoder input to solve the problem of inconsistent data distribution.
```
Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: generation.In -> generation. In

Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: stage -> **stages**

**Review by abhinavarora — CHANGES_REQUESTED** [review]

File: scheduled_sampling/README.en.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+
+In the process from a standard sequence to a sequence model, if an incorrect element is created in the preceding part of the sequence , the subsequent input state will be affected, and the error will gradually accumulate as the generation proceeds forwards. Scheduled Sampling uses the generated element as a decoder input with a certain probability, so that even if the previous one generates error, its training target is still to maximize the probability of the real target sequence, and the model will do the training in the right direction. Therefore, this method increases the fault-tolerant capability of the model.
```
Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: forwards -> **forward**

File: scheduled_sampling/README.en.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+
+## Algorithm Introduction
+
+Scheduled Sampling is mainly used in the training stage from sequence to sequence model.But It doesn't require Scheduled Sampling in  generation phrase
```
Comment (abhinavarora) [inline scheduled_sampling/README.en.md]: model.But -> model. But

**Review by abhinavarora — CHANGES_REQUESTED** [review]

**Comment by shanyi15** [comment]
@vienous please polish this doc according to the reviewer's suggestion, thank you!

**Comment by skylarch** [comment]
To make things easier for you, I made a separate pull request with the reviewed version of the translation. Please close this pull request, and thanks for your contributions.

**Pull request closed without merging**